src/plugin.ts

```diff
@@ -76,6 +76,15 @@ const recommendedRules = {
 	],
 } satisfies Linter.RulesRecord;
 
+const stylisticRules = {
+	...Object.fromEntries(
+		Object.entries(rules)
+			// eslint-disable-next-line @typescript-eslint/no-deprecated
```

Comment:
I asked in the ESLint discord if there's a recommended alternative to this prop.  This is the same prop that `eslint-plugin-eslint-plugin` is using for including in its various configs.